lib/blockchain.py

```diff
@@ -420,6 +423,37 @@ def get_suitable_block_height(self, suitableheight, chunk=None):
 
         return blocks1['block_height']
 
+    _cached_asert_anchor: Optional[asert_daa.Anchor] = None  # cached Anchor, per-Blockchain instance
+    def get_asert_anchor(self, prevheader, mtp, chunk=None):
+        if networks.net.asert_daa.anchor is not None:
+            # Checkpointed (hard-coded) value exists, just use that
+            return networks.net.asert_daa.anchor
```

Comment:
My guess is that probably this won't be how we do the hardcoding, rather we will just hardcode the values of `anchor.bits`, `anchor.prev_time`, `anchor.height`, and the function `get_asert_anchor()` will cease to exist.

Actually a fair bit of simplifying refactoring can be done once we pass that date, so I'd just recommend to leave this 'future plan' thing out for now.

Reply:
Well this code is "ready" for that eventuality since anchor really is a tuple of those values... but yes. I suspect once we checkpoint the anchor block this function can disappear 100% and we just directly slurp up that value from the EC equivalent of `chainparams`: `networks.net`.  Correct.